### ISSUE TITLE
is-valid-autocomplete missing token

### ISSUE THREAD

**Issue by Mottie** [dequelabs/axe-core#3854]
### Product

axe-core

### Product Version

4.5.2

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation

Allow "webauthn" as a token in the `autocomplete` attribute with a maximum of 5 tokens.

#### Actual

Shows an "autocomplete attribute must be used correctly" issue

#### How to Reproduce

Test with "webauthn" in the autocomplete value

```html
<input required autocomplete="webauthn work email" type="email" name="email" id="work_email">
```

#### Additional context

- [webauthn in HTML standard](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete-webauthn)
- [table showing 5 token max](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#determine-a-field's-category)
- It's difficult for me to make sense of the spec, but "the user agent should show [public key credentials](https://w3c.github.io/webauthn/#public-key-credential) available via [conditional](https://w3c.github.io/webappsec-credential-management/#dom-credentialmediationrequirement-conditional) mediation when the user interacts with the form control" seems to include values like "silent", "required", "conditional", etc?
- webauthn is only valid for `input` and `textarea` elements.

Note: I don't have a need for this addition, it's just something I noticed while trying to figure out if `section-` can be followed by any text.

**Comment by WilcoFiers** [comment]
`webauthn` is new in HTML. It wasn't there when we built this. Thank you for sharing. We'll be sure to update.

**Comment by padmavemulapati** [comment]
Validated with the latest axe-core develop branch code base, 
`autocomplete-valid` rule now allowing for `webauthn` which was not implemented earlier.
Here `auto-complete-valid` rule is failing when we pass `webauthn` as `webauthn work email`
Passing for the below webauthn tokens
```
autocomplete="email webauthn" 
 autocomplete="section-foo email webauthn" 
 autocomplete="section-foo work email webauthn" 
 autocomplete="section-foo billing work email webauthn" 
```
![image](https://user-images.githubusercontent.com/47409579/214506857-2d5ccae2-0dc5-4054-8416-36b98f35eec0.png)
